relate/templates/base.html

```diff
@@ -59,6 +51,13 @@
   </head>
 
   <body>
+    {% block lte_ie9_warning %}
+      <!--[if lte IE 9]>
+        <div class="browser alert alert-danger warning"><i class="fa fa-warning"></i>
+          {% trans "This website may not be compatible with your outdated Internet Explorer version. If you want use Internet Explorer, please install one with version higher than 10." %}
```

Comment:
"higher than 10" -> "10 or higher"?

Reply:
You're right. Can you correct that? thanks.

Reply:
Done

Reply:
Thank you!